### ISSUE TITLE
How to convert Google Colab notebook into python modular programming as individual file scripts

### ISSUE THREAD

**Issue by AGRocky** [DevashishPrasad/CascadeTabNet#176]
Hi Folks, greetings of the day to y'all.
I have successfully tried to run the code in the notebook with adding few other functionality to it  and it works fantastic on colab. 
However, now I want this to run on my local machine with VScode IDE as modules. Please guide me through this. It would be deeply appreciated.

**Comment by linkstatic12** [comment]
Can you please share the code?

**Comment by Quanult** [comment]
Hi @AGRocky, I have problems with running on the local host too, maybe we can help each other. Where are you get stuck?

**Comment by AGRocky** [comment]
@Quanult Hi thanks for the reply. I am able to run it on google colab but no idea how to integrate it as modular structure in VScode

**Comment by Quanult** [comment]
@AGRocky yes, i know it, but you have to tell me what is your problem is so I can discuss with you about that. The setup, installation or something? :D

**Comment by AGRocky** [comment]
for the installation of packages it doesn't take the versions

**Comment by Quanult** [comment]
try following this issue [166](https://github.com/DevashishPrasad/CascadeTabNet/issues/166)

**Comment by linkstatic12** [comment]
this doesn't install mmcv. it fails on the setup.py
error: subprocess-exited-with-error
  
  python setup.py bdist_wheel did not run successfully.
  exit code: 1
  
  [728 lines of output]
  C:\Users\immor\anaconda3.0\envs\test\lib\site-packages\setuptools\installer.py:30: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
    SetuptoolsDeprecationWarning,
